crates/sncast/tests/integration/fee.rs

```diff
@@ -184,8 +186,8 @@ async fn test_strk_fee_get_max_fee_with_max_gas() {
 
     let args = FeeArgs {
         fee_token: Some(FeeToken::Strk),
-        max_fee: Some(MAX_FEE.into()),
-        max_gas: Some(1_000_000_u32.into()),
+        max_fee: Some(Felt::from(MAX_FEE).try_into().unwrap()),
+        max_gas: Some(Felt::from(1_000_000_u32).try_into().unwrap()),
```

Comment:
Why are you using `NonZeroU64::new` in some test cases and `Felt::from...` in other cases for the same field? Let's pick a consistent behavior if it's possible.

Reply:
It's because fee-related fields in `FeeArgs` are use `NonZeroFelt` while in `FeeSettings::Strk/Eth` they use `NonZeroFelt`, `NonZeroU64` or `NonZeroU128`.